src/ftof.cpp

```diff
@@ -0,0 +1,59 @@
+/*! \file ftof.cpp
+ *
+ * \brief Contains functions to transfer EigenMatrix and EigenVector to FASP
+ *  or to tranfer FASP to Fenics
+ */
+
+#include "ftof.h"
+
+using namespace dolfin;
+
+// Link the dolfin::EigenMatrix Mat_A to dCSRmat format from FASP
+void EigenMatrixTOdCSRmat( dCSRmat* dCSR_A, const EigenMatrix* mat_A)
```

Comment:
I'm not sure if one way is better than the other, but is there any reason why you chose to pass a pointer to the ``dCSRmat`` rather than return it?
This could make test_ftof.cpp easier to read.
```
dCSRmat EigenMatrixTOdCSRmat(const EigenMatrix* mat_A) {
  dCSRmat dCSR_A;
  // do some magic
  return dCSR_A;
}
```